test/inductor/test_cudagraph_trees.py

```diff
@@ -99,6 +100,7 @@ def all_live_blocks():
     def all_live_block_count():
         return len(all_live_blocks())
 
+    @patch.object(torch._dynamo.config, "dynamic_shapes", False)
```

Comment:
This is a strange one, I would expect cuda graph trees to work even if `dynamic_shapes=True`, so long as everything is static.

Reply:
This needs a separate PR, I am working on that in another branch.